### ISSUE TITLE
Command-line options have to come after file(s)?

### ISSUE THREAD

**Issue by mwpastore** [ember-template-lint/ember-template-lint#1216]
This is strange:

```sh
$ node_modules/.bin/ember-template-lint --json app/templates/shop.hbs 
internal/fs/utils.js:230
    throw err;
    ^

Error: EAGAIN: resource temporarily unavailable, read
    at Object.readSync (fs.js:523:3)
    at tryReadSync (fs.js:348:20)
    at Object.readFileSync (fs.js:385:19)
    at lintFile (/Users/mwp/Code/pow/juliette/node_modules/ember-template-lint/bin/ember-template-lint.js:15:19)
    at run (/Users/mwp/Code/pow/juliette/node_modules/ember-template-lint/bin/ember-template-lint.js:156:20)
    at Object.<anonymous> (/Users/mwp/Code/pow/juliette/node_modules/ember-template-lint/bin/ember-template-lint.js:184:3)
    at Module._compile (internal/modules/cjs/loader.js:1158:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
    at Function.Module._load (internal/modules/cjs/loader.js:901:14) {
  errno: -35,
  syscall: 'read',
  code: 'EAGAIN'
}
$ node_modules/.bin/ember-template-lint app/templates/shop.hbs --json
{
  "app/templates/shop.hbs": [
    {
      "fatal": true,
      "severity": 2,
      "filePath": "app/templates/shop.hbs",
      "moduleId": "app/templates/shop",
      "message": "Parse error on line 1:\n{{each this.foo as |bar|}}{{outlet}\n----------------^\nExpecting 'CLOSE', 'OPEN_SEXPR', 'ID', 'STRING', 'NUMBER', 'BOOLEAN', 'UNDEFINED', 'NULL', 'DATA', got 'OPEN_BLOCK_PARAMS'",
      "source": "Error: Parse error on line 1: <!-- SNIP -->
```

For reference:

```
$ node --version
v12.16.1
$ node_modules/.bin/ember-template-lint --version
2.4.0
```

**Comment by rwjblue** [comment]
This has to be caused by the migration to `yargs` that landed in #1196 (for 2.4.0).

**Comment by rwjblue** [comment]
Released in https://github.com/ember-template-lint/ember-template-lint/releases/tag/v2.5.2
